bioconda_utils/build.py

```diff
@@ -125,17 +125,15 @@ def build(recipe,
 
     res = pkg_test.test_package(pkg_path)
 
-    if res.returncode == 0:
+    if (res.returncode == 0) and (res.stdout.find('Unexpected exit code') == -1):
```

Comment:
So, there are cases where the mulled test fails, but the exitcode is 0? How come? Will that be fixed within mulled?